device/esp_tinyusb/test/local/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.22)
+project(libusb_test
```

Comment:
Yes, but ATM I stuck with the problem, that I can't open a device from python. I have tried to solve it and then decided to write it in C, because it was faster and it worked. 

Ok, will do that, but I need some time to understand why I can't open device from python.

Reply:
I see. I'd like to merge this soon so we can release esp_tinyusb 1.4.4. Please rebase on top of #17 .

In case you can't make python work, we can merge it without it, but please follow-up with test refactor ASAP

Reply:
@tore-espressif 
Rebase done.
I also have added the "hot_disconnect" manual test, to verify the problem with halting the device while disconnect happens during ongoing transfer.
Firmware added, but software (pytest_usb_device_hot_disconnect.py) part is not. I tried to implement the host part (opening port, receive data, re-opening port after exception while device is gone), but it is not so trivial. I will check how it is done in esp-idf monitor and try to come up with something (1). 

To run esp_tinyusb/device tests in pytest correctly, tinyusb_uninstall function need to be added to the tests, so we could launch them one after another (2). 

And rewrite device qualifier and other speed descriptor test to python. (3)

I will add them to the plan, one thing is left before it can be merged - make a decision about the LOG level in tinyusb.